webapp/components/admin_console/mfa_settings.jsx

```diff
@@ -86,7 +86,7 @@ export default class MfaSettings extends AdminSettings {
                     helpText={
                         <FormattedHTMLMessage
```

Comment:
I think it's fine coming from PM, but all developers should be updating to the preferred method when they come across them